### PR TITLE
ORLP - 441(Deck&DeckComment)

### PR DESCRIPTION
**Pull request by serhiivenhrynovych**

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java [hunk 1]
```diff
@@ -21,80 +21,61 @@
...
     }
 
     @Auditable(action = AuditingAction.CREATE_COMMENT_FOR_DECK)
-    @PostMapping(value = "/api/category/{categoryId}/deck/{deckId}/comment")
-    @PreAuthorize(value = "@accessToUrlService.hasAccessToDeck(#categoryId)")
-    public ResponseEntity<CommentDTO> addCommentForDeck(@Validated @RequestBody ReplyToCommentDTO replyToCommentDTO,
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public CommentDTO addCommentForDeck(@Validated @RequestBody ReplyToCommentDTO replyToCommentDTO,
                                                         @RequestBody String commentText,
```
Comment (MoGUREV) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java]: It's seems, that this variable is unused.

File: src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java [hunk 2]
```diff
@@ -21,80 +21,61 @@
...
-                                                        @PathVariable Long categoryId,
                                                         @PathVariable Long deckId) throws NotAuthorisedUserException {
         LOGGER.debug("Added comment to deck with id: {}", deckId);
         DeckComment comment = commentService
                 .addCommentForDeck(deckId, replyToCommentDTO.getCommentText(), replyToCommentDTO.getParentCommentId());
```
Comment (MoGUREV) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java]: I'm not sure, but maybe better to pass whole object `replyToCommentDTO`, not parts of it.

Comment (serhiivenhrynovych) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java]: I tried did like this, but it looks less readable in serviceImpl

File: src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java (hunk 1 above)
Comment (serhiivenhrynovych) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java]: got it

**Pull request merged**